html/checkout.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<body>
+
+<script>
+  (function (o, n, e, p, a, y) {
+    var s = n.createElement(p);
+    s.type = "text/javascript";
+    s.src = e;
+    s.onload = s.onreadystatechange = function () {
+      if (!o && (!s.readyState
+        || s.readyState === "loaded")) {
+        y();
+      }
+    };
+    var t = n.getElementsByTagName("script")[0];
+    p = t.parentNode;
+    p.insertBefore(s, t);
+  })(false, document, "//rawgit.com/TransbankDevelopers/transbank-sdk-js-onepay/feat/modal-iframe/lib/checkout.onepay.min.js",
```

Comment:
If we merge into master this still going to point to the branch.

I think you should make this url relative to the iframe html and solve the problem